### PR TITLE
upgrade(covalent): upgrade to @covalent@1.0.0-beta.8 -> @1.0.0-beta.8.1

### PR DESCRIPTION
**Pull request by richavyas**
## Description

Upgrade dependencies to @covalent@1.0.0-beta.8 & 1.0.0-beta.8.1

### What's included?

- Pointing to @covalent@1.0.0-beta.8 => 1.0.0-beta.8.1
- Upgraded material & cdk to @1.0.0-beta.11 => @1.0.0-beta.12
- Upgraded to angular-material to angular@4.4.3

#### Test Steps

- [ ] rm - rf node_modules
- [ ] yarn i
- [ ] Run on browser to see if everything is working fine

##### Screenshots or link to CodePen/Plunker/JSfiddle

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,36 +37,36 @@
...
-    "@covalent/highlight": "1.0.0-beta.7",
-    "@covalent/http": "1.0.0-beta.7",
-    "@covalent/markdown": "1.0.0-beta.7",
+    "@angular/cdk": "2.0.0-beta.11",
+    "@angular/material": "2.0.0-beta.11",
+    "@angular/platform-browser": "4.4.3",
+    "@angular/platform-browser-dynamic": "4.4.3",
+    "@angular/platform-server": "4.4.3",
+    "@angular/router": "4.4.3",
+    "@covalent/core": "1.0.0-beta.8",
```
Comment (kyleledbetter) [inline package.json]: you can go straight to `1.0.0-beta.8-1` for all the covalent dependencies in this PR

File: src/app/dashboard-product/features/features.component.html [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+        <mat-list class="will-load">
             <div class="md-padding" *ngIf="!filteredFeatures || filteredFeatures.length === 0" layout="row" layout-align="center center">
                 <h3>No feature to display.</h3>
             </div>
             <ng-template let-item let-last="last" ngFor [ngForOf]="filteredFeatures">
-                <md-list-item>
-                    <md-slide-toggle mdListIcon [(ngModel)]="item.enabled"></md-slide-toggle>
-                    <h3 mdLine> {{item.title}} </h3>
+                <mat-list-item>
+                    <md-slide-toggle matListIcon [(ngModel)]="item.enabled"></md-slide-toggle>
```
Comment (kyleledbetter) [inline src/app/dashboard-product/features/features.component.html]: `md-slide-toggle` needs to be updated to `mat-slide-toggle`

File: src/app/dashboard-product/dashboard-product.component.html [hunk 3]
```diff
@@ -1,69 +1,69 @@
...
-    <md-icon>menu</md-icon>
+  <button mat-icon-button td-menu-button tdLayoutToggle>
+    <mat-icon>menu</mat-icon>
   </button>
   <div td-toolbar-content layout="row" layout-align="center center" flex>
     <span>Quickstart</span>
     <span flex></span>
-    <a md-icon-button mdTooltip="Docs" href="https://teradata.github.io/covalent/" target="_blank"><md-icon>chrome_reader_mode</md-icon></a>
-    <a md-icon-button mdTooltip="Github" href="https://github.com/teradata/covalent" target="_blank"><md-icon svgIcon="assets:github"></md-icon></a>
+    <a mat-icon-button mdTooltip="Docs" href="https://teradata.github.io/covalent/" target="_blank"><mat-icon>chrome_reader_mode</mat-icon></a>
```
Comment (kyleledbetter) [inline src/app/dashboard-product/dashboard-product.component.html]: `mdTooltip e` needs to be updated to `matTooltip `

File: src/app/dashboard-product/features/features.component.html [hunk 4]
```diff
@@ -1,35 +1,35 @@
...
+                    <h3 matLine> {{item.title}} </h3>
                     <span flex></span>
                     <span flex-gt-xs="60" flex-xs="40" layout-gt-xs="row">
                         <div class="md-caption tc-grey-500" flex-gt-xs="50"> {{ item.user }} </div>
                         <div class="md-caption tc-grey-500" flex-gt-xs="50"> {{ item.modified | timeAgo }} </div>
                     </span>
                     <span>
-                        <button md-icon-button [md-menu-trigger-for]="menu">
-                        <md-icon>more_vert</md-icon>
+                        <button mat-icon-button [md-menu-trigger-for]="menu">
```
Comment (kyleledbetter) [inline src/app/dashboard-product/features/features.component.html]: `md-menu-trigger-for` needs to be `matMenuTriggerFor `

File: src/app/dashboard-product/features/features.component.html [hunk 5]
```diff
@@ -1,35 +1,35 @@
...
+                        <mat-icon>more_vert</mat-icon>
                         </button>
                         <md-menu x-position="before" #menu="mdMenu">
```
Comment (kyleledbetter) [inline src/app/dashboard-product/features/features.component.html]: `md-menu` -> `mat-menu`
`mdMenu` -> `matMenu`
`md-menu-item` -> `mat-menu-item`

**Comment by kyleledbetter** [comment]
Obviously still `in progress` but seems like you couldn't saved work by using https://www.npmjs.com/package/angular-material-prefix-updater

**Comment by emoralesb05** [comment]
I was about to say... why not use the tool ?

**Pull request merged**